.ddev/.setup/scripts/utils.sh

```
@@ -0,0 +1,563 @@
+        export TYPO3_BIN="$BASE_PATH/vendor/bin/typo3"
+    fi
+    mysql -uroot -proot -e "DROP DATABASE IF EXISTS $DATABASE"
+}
+
+# Function to create symlinks for the main extension.
+# It iterates over the items in the current directory, excluding certain directories and files,
+# and creates symbolic links for the remaining items in the specified base path.
+function create_symlinks_main_extension() {
+    local exclusions=(".*" "Documentation" "Documentation-GENERATED-temp" "var")
+    for item in ./*; do
+        local base_name=$(basename "$item")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)